powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/OperationCreateRequest.java

```diff
@@ -1,48 +1,30 @@
-/*
- * PowerAuth Server and related software components
- * Copyright (C) 2020 Wultra s.r.o.
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as published
- * by the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
 package com.wultra.security.powerauth.client.model.request;
 
 import com.fasterxml.jackson.annotation.JsonSetter;
 import com.fasterxml.jackson.annotation.Nulls;
 import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.*;
 import lombok.Data;
 
 import java.util.*;
 
-/**
- * Request method for creating a new operation.
- *
- * @author Petr Dvorak, petr@wultra.com
- */
 @Data
 public class OperationCreateRequest {
 
     @Schema(description = "The identifier of the user", requiredMode = Schema.RequiredMode.NOT_REQUIRED)
+    @Size(min = 1, message = "User ID must not be empty when creating operation")
```

Comment:
Is there any logic in choosing `@Size(min = 1)` versus `@NotBlank`?

Reply:
`@NotBlank` does not allow null.

https://www.baeldung.com/java-bean-validation-not-null-empty-blank#a-side-by-side-comparison